src/zenml/orchestrators/local/local_orchestrator.py

```diff
@@ -111,33 +114,30 @@ def run_pipeline(
 
         # Run each component. Note that the pipeline.components list is in
         # topological order.
-        for node in pipeline_proto.nodes:  # type: ignore[attr-defined]
-            context = node.pipeline_node.contexts.contexts.add()
+        for node in pb2_pipeline.nodes:  # type: ignore[attr-defined]
+            pipeline_node: PipelineNode = node.pipeline_node  # type: ignore[valid-type]
+            context = (
+                pipeline_node.contexts.contexts.add()  # type: ignore[attr-defined]
+            )
             context_utils.add_stack_as_metadata_context(
```

Comment:
```
context_utils.add_stack_as_metadata_context(context=context, stack=stack)
context_utils.add_runtime_configuration_to_node(runtime_configuration, pipeline_node)
```
Even though these two functions are similar in terms of functionality, they look a bit different with their input space. Would it make sense to refactor it in a way where we would pass the `pipeline_node` to the first function instead of `context` for the sake of consistency?